### PR TITLE
Return XML instead of DOMDocument object when using _serialize viewVar

### PR DESCRIPTION
**Pull request by rlems**
Fix for #7897

### CODE REVIEW

File: src/View/XmlView.php [hunk 1]
```diff
@@ -141,6 +141,9 @@ protected function _serialize($serialize)
             $options['pretty'] = true;
         }
 
+        if(isset($options['return']) && strtolower($options['return']) === 'domdocument') {
```
Comment (markstory) [inline src/View/XmlView.php]: Should be `if (`.

**Comment by markstory** [comment]
Any way we could get an updated test case?

**Comment by markstory** [comment]
The pull request you've made is for 3.x, but the issue you reported was on 2.x. Did you intend to apply a fix for 2.x?

**Comment by rlems** [comment]
For the test case: I don't have a lot of experience with that, so I'll read up on it and I'll probably get those done next week.
For the version: I wanted to do it to both versions, but I guess I'll have to make the fix for both then?

**Comment by markstory** [comment]
@rlems If you can get the fix done for one of the versions, we can take care of porting the changes to the other branch.

**Pull request merged**